### PR TITLE
Add git-annex-remote-rclone

### PR DESCRIPTION
**Pull request by jwodder**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

---

@yarikoptic @danieldent
* Please comment that you are willing to be listed as maintainers
* Tests are needed for this package

### CODE REVIEW

**Comment by yarikoptic** [comment]
Willing

**Comment by yarikoptic** [comment]
@jwodder for tests -- please take this script https://github.com/DanielDent/git-annex-remote-rclone/pull/45/files#diff-945023c48e6c45e72aa3374651a98f29c93c57bcb40e02f1b44a5cb53b03e0b2  (may be would need to disable testing for "test 0" case)

**Comment by DanielDent** [comment]
Willing

**Comment by yarikoptic** [comment]
Dear @jwodder, please update it for already released 0.7 and ensure that CI is happy so conda people could consider merging this recipe.  Thank you in advance!

**Comment by jwodder** [comment]
@conda-forge/staged-recipes Ready for review.

File: recipes/git-annex-remote-rclone/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/DanielDent/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a119e3a4c194ff24a07aa1620714d60e0dc2a3148f5479cb07faf31e0e31889c
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: mkdir -p $PREFIX/bin && cp git-annex-remote-rclone $PREFIX/bin
```
Comment (ocefpaf) [inline recipes/git-annex-remote-rclone/meta.yaml:15]: Is this a repackage? Can we build it from source?

Comment (jwodder) [reply recipes/git-annex-remote-rclone/meta.yaml:15]: It's a bash script that we're acquiring directly from the GitHub repository.

Comment (yarikoptic) [reply recipes/git-annex-remote-rclone/meta.yaml:15]: the entire project is just a bash script - so nothing to build etc. `cp` should be good/enough

**Pull request merged**